### ISSUE TITLE
exclude_PCA_corr is ignored regardless of value

### ISSUE THREAD

**Issue by aofarrel** [UW-GAC/analysis_pipeline_cwl#2]
## Setup in SB
<img width="474" alt="Screenshot 2021-05-10 at 4 37 06 PM" src="https://user-images.githubusercontent.com/27784612/117738038-f56ebb00-b1af-11eb-98d3-2c27e40fc6b7.png">

## Resulting config file on SB
<img width="644" alt="Screenshot 2021-05-10 at 4 36 53 PM" src="https://user-images.githubusercontent.com/27784612/117738042-f7387e80-b1af-11eb-9044-33e0ada903c3.png">

## Result
On testdata inputs, this is a significant enough difference for the output to change, albeit not everywhere. For example, using my WDL's outputs as truth files, exclude_pca_corr not showing up the configuration file would have the resulting RData (and derived subset gds files) outputs still match for chr1 and chr3, but not for chr2. Of course, the merged GDS outcomes don't match either, but the actual differences only show up in a few places.

## Possible Cause
The CWL implements this variable like so:
```
if(inputs.exclude_pca_corr)
              cmd_line += "exclude_pca_corr " + inputs.exclude_pca_corr + "\n"
```
If I had to guess, this is why it's not showing up in the CWL.
* When exclude_pca_corr is not defined, which effectively acts as if exclude_pca_corr equalled true in regard to the R script and therefore isn't gonna be set if the user wants it to be true, if(inputs.exclude_pca_corr) evaluates to false
* When exclude_pca_corr is defined as false, if (inputs.exclude_pca_corr) still evaluates to false because exclude_pca_corr itself is false

**Comment by smgogarten** [comment]
Fixed in c2eb59b by setting default value in tool

**Comment by aofarrel** [comment]
To confirm: It seems that now that if it is set to false, exclude_pca_corr is not getting written to the config file at all. Is that the intent?
